common/src/main/java/io/druid/audit/AuditInfo.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Druid - a distributed column store.
+ * Copyright 2012 - 2015 Metamarkets Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.druid.audit;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AuditInfo
+{
+  private final String author;
+  private final String emailId;
+  private final String comment;
+
+  @JsonCreator
+  public AuditInfo(
+      @JsonProperty("author") String author,
+      @JsonProperty("emailId") String emailId,
```

Comment:
author and email seem redundant, their both some kind of user ID. Author seems sufficient to me for this purpose. If someone needs more information than can maintain a separate table and enforce foreign key relation at the database level.